documentation/List-of-Action-plugins.md

```diff
@@ -535,6 +543,34 @@ The output object `notification` contains all the fields from the notification r
 
 `id, user_id, application_id, review_id, email_recipients, subject, message, attachments, email_sent, is_read`
 
+---
+
+### Schedule Action
+
+A "special" action that allows other actions to be triggered at some time in the future, which can be used to expire licenses, registrations, or notify of upcoming deadlines.
+
+- _Action Code:_ **`scheduleAction`**
+
+| Input parameters<br />(\*required) <br/> | Output properties |
+| ---------------------------------------- | ----------------- |
+| `duration`\*                             | `scheduledEvent`  |
+| `eventCode`                              |                   |
+| `applicationId`                          |                   |
+| `templateId`                             |                   |
+| `cancel`                                 |                   |
+| `data`                                   |                   |
+
+This Action stores an "event" in the database `trigger_schedule` table, scheduled for a time in the future specified by `duration`. When this time is reached, a special trigger is fired (`ON_SCHEDULE`) which can be used as the trigger for subsequent actions.
+
+Each scheduled event can be saved with an `eventCode` -- this is used by Actions that are triggered by this event to determine *which* action should be fired for any given `ON_SCHEDULE` trigger on each template type. Every Action defined for each template has an optional `scheduledActionCode` field, which can be used to match specific events. If no event code is provided, then *every* action for that template type with an `ON_SCHEDULE` trigger will be executed.
+
+By default, when an event is saved, the `outputCumulative` object from the `scheduleAction` action (including outputs of previous actions in the sequence) is stored in the `data` field, but additional data can be added to this as well. Then when the scheduled action runs, it gets this data passed in as `outputCumulative`, so it has access to the output info of previous actions -- this means it effectively can continue as though it were part of the same sequence of actions, but with a "pause" due the the schedule.
+
+The `cancel` parameter is a way to prevent a previously scheduled event from occurring. Any existing event that has matching `applicationId` and `eventCode` will be set to inactive without ever firing. In practice, though, targeting an event by `applicationId` is often not feasible, so the preferred way to cancel a scheduled event is to just apply an appropriate Condition to the subsequent action -- so the event is still triggered, but the matching action won't occur if the condition is not met (e.g. don't expire a product if registration has been renewed)
```

Comment:
```suggestion
The `cancel` parameter is a way to prevent a previously scheduled event from occurring. Using the action to run with the cancel will find existing event that has matching `applicationId` and `eventCode` will be set to inactive without ever firing. In practice, though, targeting an event by `applicationId` is often not feasible, so the preferred way to cancel a scheduled event is to just apply an appropriate Condition to the subsequent action -- so the event is still triggered, but the matching action won't occur if the condition is not met (e.g. don't expire a product if registration has been renewed)
```